src/handlers/dataProvidersHandlers.ts

```diff
@@ -83,6 +84,20 @@ const putDataProvider = (
 		);
 		return;
 	}
+	const { keycloakOrganizationId } = req.params;
+	if (
+		keycloakOrganizationId !== undefined &&
+		keycloakOrganizationId !== null &&
+		!isKeycloakId(keycloakOrganizationId)
+	) {
+		next(
+			new InputValidationError(
+				'Invalid Keycloak organization id.',
+				isKeycloakId.errors ?? [],
+			),
+		);
+		return;
```

Comment:
Mind adding a few tests around requests with undefined / null / non-uuid `keycloakOrganizationId`?

Reply:
I shall.

Reply:
After looking again, I think we have ample tests for Data Provider, but we needed to strictify some of them. For example, with `toMatchObject` didn't care whether the key was there with null or not, but `toEqual` or `toStrictEqual` gave the expected failures if the key was not present.